fl4health/privacy_mechanisms/dpPoissonBinomial.py

```diff
@@ -0,0 +1,162 @@
+import math
+from typing import List
+
+import torch
+from scipy.stats import ortho_group
+from torch.distributions import Binomial
+
+
+def ScalerPoissonBinomial(x: float, N: int, radius: float, theta: float) -> int:
+    """
+    Privacy mechanism which encodes scalar data x as a bionomial random variable.
+
+    Args:
+        x (float): Scalar data to be encoded.
+        N (int): Number of Bernoulli trials that yields the binomial variable (We have N = modulus - 1).
```

Comment:
The "N=modulus -1" comment here is a bit off-hand. Is this referring to something in the paper? If so, I'd reference where.